lib/typescript/httpclient/endpoints/updateTag.ts

```diff
@@ -1,7 +1,8 @@
 import {Tag} from '../model';
 import {HttpClient} from '../client';
 
-export default HttpClient.prototype.updateTag = async function (tag: Tag) {
-  await this.doFetchFromBackend('tags.update', {...tag});
-  return Promise.resolve(true);
+export default HttpClient.prototype.updateTag = function (tag: Tag) {
+  return this.doFetchFromBackend('tags.update', {...tag})
+    .then(() => true)
+    .catch(() => false);
```

Comment:
Can't we do this in a different way? It doesn't feel right @bitboxer

Reply:
It always returned `true` before. I just changed the thing to actually return `false` if something went wrong. But I can totally refactor it and not make it return a boolean but a real promise object